File: pset1/mario/more/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    // Read input
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height > 23 || height < 0);

    for (int i = 0; i < height; i++)
    {
        // Print spaces
        for (int j = 0; j < height - 1 - i; j++)
        {
            printf(" ");
        }

        // Print hashes
        for (int j = 0; j < i + 1; j++)
        {
            printf("#");
        }

        printf("  ");

        // Print hashes
        for (int j = 0; j < i + 1; j++)
        {
            printf("#");
        }

        printf("\n");
    }
}

File: pset6/mario/less/mario.py
from cs50 import get_int

n = get_int("Height: ")
while (n > 23 or n < 0):
    n = get_int("Height: ")

for i in range(1, n + 1):
    # Prints First triangle
    print(' ' * (n - i), end='')
    print('#' * (i + 1), end=''\n')
